### ISSUE TITLE
Strange timeout behaviour

### ISSUE THREAD

**Issue by hman2** [freeotp/freeotp-android#414]
Hi,

when I want to generate a token to access the OwnCloud of employer, when I touch the key: at first the timeout is ridiculously short (10 sec or less?), so I cannot enter the token into the OwnCloud app.

The second time I touch the key the timeout is much more gentle, about 30 seconds. The third time it's fast again, then slow again and so on.

I understand that this behaviour to some extend gets controlled by the issued key, but whatever is in there should be the same all the tiime, as the key doesn't change? Also, my employer's admins tell me the never changed the setting.

Version is 2.04 (45) (to my knowledge the latest)..

Thanks
hman2

**Comment by jimmysjolund** [comment]
I have seen older issues mentioning the same issue. I can not reproduce it myself using 2.0.5. If still relevant, do you see the same behaviour in 2.0.5?

**Comment by joggee-fr** [comment]
TOTP changes typically every 30 seconds (default configuration) and it does not depend at all to the time when the code has been displayed in the app. So the described behavior may be perfectly normal depending on the time remaining in the absolute 30s time slot.

**Comment by justin-stephenson** [comment]
> TOTP changes typically every 30 seconds (default configuration) and it does not depend at all to the time when the code has been displayed in the app. So the described behavior may be perfectly normal depending on the time remaining in the absolute 30s time slot.

Yes, this can be easily tested with `Counter` (HOTP) versus `Timeout` (TOTP) radio buttons in https://freeotp.github.io/qrcode.html 

I think this can be closed.
